### ISSUE TITLE
Casting a spell at an opponent's Mirrorwing Dragon

### ISSUE THREAD

**Issue by spjspj** [magefree/mage#2081]
If an opponent controls one and you cast swords to plowshares targetting it, it should copy for all of your controlled creatures.  This is what I'm seeing on my local server:

```
Game exception occurred: java.lang.IllegalStateException: Target wasn't handled. class:class mage.abilities.effects.common.TargetWithAdditionalFilter
Server version: 1.4.13v1
mage.player.ai.ComputerPlayer.choose(ComputerPlayer.java:453)
mage.player.ai.ComputerPlayer.choose(ComputerPlayer.java:201)
mage.abilities.effects.common.CopySpellForEachItCouldTargetEffect.apply(CopySpellForEachItCouldTargetEffect.java:179)
mage.abilities.AbilityImpl.resolve(AbilityImpl.java:197)
mage.abilities.TriggeredAbilityImpl.resolve(TriggeredAbilityImpl.java:100)
mage.game.stack.StackAbility.resolve(StackAbility.java:105)
mage.game.GameImpl.resolve(GameImpl.java:1370)
mage.game.GameImpl.playPriority(GameImpl.java:1321)
mage.game.turn.Step.priority(Step.java:87)
mage.game.turn.Phase.playStep(Phase.java:210)
mage.game.turn.Phase.play(Phase.java:115)
mage.game.turn.Turn.play(Turn.java:152)
mage.game.GameImpl.playTurn(GameImpl.java:852)
mage.game.GameImpl.play(GameImpl.java:766)
mage.game.GameImpl.start(GameImpl.java:733)
mage.server.game.GameWorker.call(GameWorker.java:60)
java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
java.util.concurrent.FutureTask.run(FutureTask.java:166)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
java.lang.Thread.run(Thread.java:722)
```

**Comment by drmDev** [comment]
@fireshoes also noted this on Mirrorwing Dragon on the forums:
"When a spell I control targets only my Mirrorwing Dragon, the number of available targets for the copied spells includes my opponent's creatures, and the opponent's creature is allowed as a valid target. The copies should only be made for each other creature the caster controls.

Mirrorwing Dragon is also triggering (but not doing anything) when a player casts an instant or sorcery that doesn't target it (e.g. Wrath of God)."

**Comment by spjspj** [comment]
Fixes with 09cc97dd85f51f069ac6d3b8a5e9040618a0647b and e5117a967d8f5674fc7c9331cdb482973eaad92f

**Comment by spjspj** [comment]
I think that this is fixed in 1.4.13v2.
